<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tervetuloa</class>
 <widget class="QDialog" name="Tervetuloa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelAika">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>771</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>150</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>30</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>510</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ovi kiinni</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTervetuloa">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>771</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tervetuloa</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTime">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>69</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>20:45</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
